src/platform/forms-system/src/js/widgets/CheckboxWidget.jsx

```diff
@@ -25,6 +25,9 @@ export default function CheckboxWidget({
         required={required}
         disabled={disabled}
         onChange={event => onChange(event.target.checked)}
+        aria-describedby={
```

Comment:
Added in a similar manner to the [text widget](https://github.com/department-of-veterans-affairs/vets-website/blob/6a233f3c172ebf734a9b8f6cd065d6fa8d494a16/src/platform/forms-system/src/js/widgets/TextWidget.jsx#L37)

Reply:
haha yeah, I'm just shamelessly lifting what you threw in there lol